fsspec/implementations/http.py

```diff
@@ -103,6 +103,8 @@ def __init__(
         request_options.pop("listings_expiry_time", None)
         request_options.pop("max_paths", None)
         request_options.pop("skip_instance_cache", None)
+        request_options.pop("listings_cache_type", None)
```

Comment:
This is problematic - where else might these kwargs pollute calls? Do we need to change the signatures of all backends?